Proporcion.py

```diff
@@ -6,6 +6,73 @@
 """
     <summary>
         Funcion menu que realiza la prueba de hipotesis de una proporcion.
+    <param name="x">Numero de exitos en la muestra.</param>
+    <param name="n">Numero de ensayos en la muestra.</param>
+    <param name="alpha">Nivel de significancia.</param>
+    <param name="Z">Valor del estadistico de prueba.</param>
 """
 def menu_proporcion():
-    return
\ No newline at end of file
+    print("Elige la prueba de hipotesis a utilizar:")
+    print("1)")
+    print("H0. " + Stheta + " = " + Stheta + "0")
+    print("H1. " + Stheta + " > " + Stheta + "0")
+    print("2)")
+    print("H0. " + Stheta + " = " + Stheta + "0")
+    print("H1. " + Stheta + " < " + Stheta + "0")
+    print("3)")
+    print("H0. " + Stheta + " = " + Stheta + "0")
+    print("H1. " + Stheta + " != " + Stheta + "0")
+    opcion = input("Opcion: ")
+    os.system("cls")
+
+    if opcion not in ["1", "2", "3"]:
+        print("Opcion incorrecta, vuelve a intentarlo.")
+        print("Presiona una tecla para continuar...")
+        os.system("pause")
+        os.system("cls")
+        menu_proporcion()
+        return
+
+    print("Prueba de hipotesis de proporciones")
+    print("H0. " + Stheta + " = " + Stheta + "0")
+    print("H1. " + Stheta + [">", "<", "!="][int(opcion) - 1] + Stheta + "0")
+    print("")
+    print("Introduce los datos:")
+    theta = float(input(Stheta+"0: "))
+    x = float(input("x: "))
+    n = int(input("n: "))
+    alpha = float(input(Salpha + ": "))
```

Comment:
```suggestion
    alpha = float(input(Salpha + ": "))
    alpha = 0.5 - alpha
```

Podrías hacer esto aquí en vez de la función prueba, como harás esa misma operación en cada región crítica te puedes ahorrar y facilitar la vida con esto

Reply:
Es que en el último no podrías, habría que poner 0.5-(0.5-alpha)/2